packages/terra-folder-tree/tests/wdio/folder-tree-spec.js

```diff
@@ -91,6 +91,23 @@ Terra.describeViewports('FolderTree', ['medium'], () => {
       Terra.validates.screenshot('last item focused keyboard', { selector: '#expand-collapse-folder-tree' });
     });
 
+    it('remembers the last focused item when tabbing in and out', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/folder-tree/expand-collapse-folder-tree');
+
+      $('aria/Expand All').click();
+      browser.keys('Tab');
+      browser.keys('Tab');
+      browser.keys('ArrowDown');
+      browser.keys('ArrowDown');
+      Terra.validates.screenshot('level two folder focused keyboard', { selector: '#expand-collapse-folder-tree' });
+
+      browser.keys('Tab');
+      Terra.validates.screenshot('tabbed out of focus', { selector: '#root' });
+
+      browser.keys(['Shift', 'Tab']);
+      Terra.validates.screenshot('level two folder focused keyboard', { selector: '#expand-collapse-folder-tree' });
```

Comment:
I like that we're avoiding adding extra screenshots here, just wondering if there are any potential issues with validating the same screenshot twice in this test.

Reply:
I've done this in the past before, and both screenshots should result in the same identical image, so I believe it should be alright

Reply:
Is there a need to validate teh same screenshot twice? If we're only checking for focus, we can use the `isFocused()` validator since that's more efficient than doing a screenshot comparison.

Reply:
Oh that's right! I keep forgetting we have that, I'll update it to use that